recipes/rdock/meta.yaml

```diff
@@ -1,47 +1,60 @@
-package:
-  name: rdock
-  version: 2013.1
+{% set name = "rDock" %}
+{% set version = "0.22" %}
 
-build:
-  number: 2
-  skip: True  # [osx]
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  url: https://depot.galaxyproject.org/software/rDock/rDock_2013.1_src_all.tar.gz
-  sha256: e716998c3f8a2a70205a8d30ba22675bfdb1764d13c858645138c5eadf2a37e9
+  url: https://github.com/CBDD/rDock/releases/download/v{{ version }}/{{ name }}_2022_v0.tar.gz
+  sha256: 3dd4d4eae8cdbcfd5cec8584df43f5fc4fba2267b6c7041bf93e96705277ba62
   patches:
-    - rDock_python_fixes.patch
-    - rDock_fix_paths.patch
+    - 0001-Add-missing-cmath-for-cos-.-usage.patch
+
+build:
+  number: 0
+  skip: True  # [osx]
 
 requirements:
   build:
-    - make
+    - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - make
+    - perl *
   host:
     - cppunit
-    - perl
     - popt
-    - python
   run:
     - numpy
-    - perl
-    - popt
+    - perl *
+    - python *
     - openbabel
 
 test:
   commands:
+    - grep() { sed "/$*/{:m;n;bm};\$q1" ; }
+    - rbcalcgrid 2>&1 | grep rDock
     - rbcavity --help
     - rbdock --help
-    - sdrmsd --help
+    - rbhtfinder 2>&1 | grep "high throughput" 
+    - rblist 2>&1 | grep rDock
+    - rbmoegrid 2>&1 | grep rDock
+    - rbrms 2>&1 | grep RMSD
+    - sdfilter -h
+    - sdmodify -h
+    - sdreport -h
     - sdsort -h
-
+    - sdsplit -h
```

Comment:
TODO: The compiled binaries are put into `${PREFIX}/bin`, but `build.sh` needs to be updated to include the Perl/Python scripts.

Reply:
done